packages/react-native-editor/__device-tests__/gutenberg-editor-device-actions.test.js

```diff
@@ -27,6 +27,10 @@ describe( 'Gutenberg Editor Rotation tests', () => {
 		await editorPage.addNewBlock( blockNames.paragraph );
 
 		if ( isAndroid() ) {
+			// Waits until the keyboard is visible
+			await editorPage.driver.waitUntil( async function () {
+				return await editorPage.driver.isKeyboardShown();
+			} );
```

Comment:
Nitpick:
```suggestion
			await editorPage.driver.waitUntil(
				editorPage.driver.isKeyboardShown
			);
```

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/57460/commits/7e6fa2d0646b078f3b2a2ba143b9d3d157caa0d6